### PR TITLE
Improve screensaver management

### PR DESCRIPTION
**Pull request by jvoisin**
- Add modern ways to unlock Linux machines remotely
- Use proper `register_options`
- Clarify the actions: lock/unlock, start/stop
- Add more platforms
- Add a couple of checks before running the commands

### CODE REVIEW

**Comment by jvoisin** [comment]
I'm a bit confused by the red CI, am I doing something wrong?

File: modules/post/multi/manage/screensaver.rb
Comment (sjanusz-r7) [inline modules/post/multi/manage/screensaver.rb:58]: There's a few permutations of these commands that we would want to handle. E.g.: 
- both commands are found, but both exit with a non-zero status code; this should likely be reported to the end-user as a failure.
- The first command succeeds, in which case, do we want to run the second command?
- Neither of the commands exist

**Review by jheysel-r7 — APPROVED** [review]

**Comment by jheysel-r7** [comment]
> I'm a bit confused by the red CI, am I doing something wrong?

You're all good. We usually submit review's as `Comment`s when really, if changes are required, we should be selecting `Request Changes`. The CI tests don't run again until a reviewer as approved the changes if they were requested. This is an example of when the review was submitted with `Request Changes`.
Screenshot 2024-09-23 at 8 11 07 AM

**Comment by jheysel-r7** [comment]
# Release Notes

Adds a number of improvements to modules/post/multi/manage/screensaver.rb. A new UNLOCK action has been added. When the LOCK action is selected instead of only checking to see if `xdg-screensaver lock` exists on the target, the module will check for the presence of `qdbus`, `dbus-send` and `loginctl`. Improved error handling when running on Windows or Solaris has also been added.

**Pull request merged**
